src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -116,6 +118,46 @@ def _f1_score(model_output: str, target_output: str, *, normalize_text: bool = F
         return float(ret)
 
 
+def _precision(model_output: str, target_output: str, *, normalize_text: bool = False) -> float:
+    """
+    Given the model output and the target output, compute the precision.
+    Precision is the fraction of words in the prediction that are also found in the target output.
+    Before computing precision, we normalize the text following the QuAC protocol.
+
+    :param model_output: The output of a model that we want to evaluate.
+    :param target_output: The reference or the "ground truth" output.
+    :param normalize_text: Normalize the text before computing f1.
+    :returns: Precision.
+    """
+    if normalize_text:  # pragma: no branch
+        model_output, target_output = (_normalize_text_quac_protocol(text) for text in (model_output, target_output))
+    ret = precision(reference=set(target_output.split(" ")), test=set(model_output.split(" ")))
```

Comment:
why set and not list? we want to discard repetitions of words?

Reply:
Valid question, but set seems standard. At least that's what the [NLTK metric](https://www.nltk.org/_modules/nltk/metrics/scores.html) assumes that we use here.